toolkit/docs/CVE/CVE-Quickstart-Guide.md

```diff
@@ -0,0 +1,167 @@
+# **Document: Fixing CVE Patches for Azure Linux**
+
+This document provides a step-by-step guide to fixing CVE patches for Azure Linux. It includes identifying the CVE, locating and preparing the package, applying the patch, and submitting it for integration into Azure Linux.
+
+---
+
+## **Steps to Fix a CVE Patch**
+
+### **1. Identify the CVE**
+
+1. Navigate to the [Astrolabe Dashboard](https://brave-ocean-0baeae310.5.azurestaticapps.net/#/) to pick a CVE.
+2. Determine the Azure Linux branch for the CVE:
+   - **3.0** → Use the `fasttrack/3.0` branch.
+   - **2.0** → Use the `fasttrack/2.0` branch.
+
+---
+
+### **2. Locate the Package**
+
+1. Clone the **azlinux repository** and switch to the appropriate branch:
+   ```bash
+   git checkout fasttrack/<branch-version>
+   ```
+2. Go to the `SPECS` directory and search for the package:
+   ```bash
+   cd ~/azurelinux/SPECS
+   ls | grep <pkg-name>
+   ```
+
+---
+
+### **3. Retrieve and Prepare the Source**
+
+1. Locate the source URL in the `<pkg>.spec` file:
+   ```bash
+   less ~/azurelinux/SPECS/<pkg>/<pkg>.spec
+   ```
+2. Download the source tarball outside azlinux repo:
+   ```bash
+   wget https://azurelinuxsrcstorage.blob.core.windows.net/sources/<pkg-source>.tar.gz
+   ```
```

Comment:
The source files can be found at:
https://azurelinuxsrcstorage.blob.core.windows.net/sources/core/<file-name>

Just a heads-up, not all files are .tar.gz. Make sure to use the file name exactly as it’s listed in the .signatures file.